### ISSUE TITLE
Can't call a script named "CreateFont"

### ISSUE THREAD

**Issue by RobertBColton** [enigma-dev/enigma-dev#1119]
I know this is a duplicate of #915 but it looks like we never fixed that specific conflict. CreateFont is a macro in Win32 apparently like all of the other functions which maps to either CreateFontA or CreateFontB. I can create a script called this in ENIGMA but if I attempt to call it I get:

```
Line 1, position 19 (absolute 18): Too few arguments to function `CreateFontA': provided 1, required 14.
```

I am not calling CreateFontA, I am calling CreateFont first of all. We should just fix this.

**Comment by JoshDreamland** [comment]
There's actually an easy solution: ***stop including `windows.h` from `SHELLmain`!***

Those are macros. They are scopeless. There is nothing we can do except stop including so many damn system-specific headers in headers included by `SHELLmain.cpp`.

**Comment by mikeypro83** [comment]
Can we close this? This is no longer a bug.

**Comment by RobertBColton** [comment]
Thanks mikey, but I am actually still able to reproduce this on master, I've been meaning to clean up the `windows.h` includes for this. I've been prioritizing higher level tasks in RadialGM and everything right now. Perhaps you were testing with SDL?

![CreateFont Script Conflicts With Win32 Macro](https://user-images.githubusercontent.com/3212801/45060179-e2090a80-b06c-11e8-84f2-4183629a23e6.png)

**Comment by mikeypro83** [comment]
ooh my gosh! you are right! sorry! yeah I forgot about that when I ran tests. uhg! it's even worse, I'm not a Windows user. lol. I kicked windows last year because of many reasons but mainly because it complicates development. sorry about that! caught up in my own world I guess.

**Comment by RobertBColton** [comment]
No need to apologize, it's always better to discuss issues openly and state facts and observances so things don't get swept aside. Thanks for your input and I will try to get this fixed soon.

**Comment by RobertBColton** [comment]
Closing as resolved by #1445, now everybody remember! Do not include `windows.h` from a header that defines user namespace stuff and gets included into `SHELLmain.cpp` or this will come back!
